app/views/caretaker_notifier_mailer/inform.html.erb

```diff
@@ -1,2 +1,2 @@
 Your friend, <%= @user.first_name %>, wants you to watch over their plants.
-Click the link to become a garden caretaker *some link here*
+<%= link_to "Caretake #{@garden.name.capitalize}", caretaker_start_url(@garden) %>
```

Comment:
So when a user clicks on the link in the email, assuming they are not already logged in to our site, will it first prompt them to log in with OAuth? 
Discussion with @bendelonlee - no, it will likely redirect them to a 404 error. Ben is working on a separate PR to fix this by having the user first create an account.

Reply:
Also similar to email - wording is a bit awkward - change to "Take care of <garden name>" maybe?